.github/workflows/test-cygwin.yml

```
@@ -0,0 +1,53 @@
+name: Test Cygwin
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: windows-2019
+    name: Cygwin Python 3.8
+
+    steps:
+      - name: Checkout Pillow
+        uses: actions/checkout@v2
+      - name: Set up Cygwin
```

Please separate each step with a newline, like the other workflows:
```suggestion

      - name: Set up Cygwin
```

---

Done